java/test/jmri/implementation/DefaultConditionalTest.java

```diff
@@ -29,7 +29,8 @@ public class DefaultConditionalTest {
      */
     @Test
     public void createInstance() {
-        new DefaultConditional("IXIC 0");
+        DefaultConditional t = new DefaultConditional("IXIC 0");
+        Assertions.assertNotNull(t);
```

Comment:
Technically, a constructor can never fail and return null.  It'll throw rather than do that.  Yes, I understand we have a huge number of these checks, but that's for historical reasons....